src/spdx_tools/spdx3/writer/console/console.py

```diff
@@ -1,21 +1,29 @@
 # SPDX-FileCopyrightText: 2023 spdx contributors
 #
 # SPDX-License-Identifier: Apache-2.0
-
+from enum import Enum
 from typing import Optional, TextIO, Union
 
 
-def write_value(tag: str, value: Optional[Union[bool, str]], out: TextIO, indent: bool = False):
+def write_value(tag: str, value: Optional[Union[bool, str, dict, list, Enum]], out: TextIO, indent: bool = False):
     """This function is duplicated from spdx_tools.spdx.writer.tagvalue.tag_value_writer_helper_functions
     and slightly adapted to make indentation of output possible."""
     if not value:
         return
-    if indent:
-        out.write(f"\t{tag}: {value}\n")
-    else:
-        out.write(f"{tag}: {value}\n")
+    if isinstance(value, dict):
+        for key, val in value.items():
+            write_and_possibly_indent(f'{tag}: "{key}": "{val}"', indent, out)
```

Comment:
You put `"` here to separate the dict output from the other tag-value combination, right? Probably more a matter of taste but I would rather print key-value pairs, so `({key}, {value})` here to avoid these `"`, but this is completely up to you.

Reply:
Sure, I don't have strong feelings on this one.

Reply:
Do you think we should have it as a list of tuples then instead of every entry in a new line?

Reply:
I don't think this makes any difference. Having a list of tuples would be consistent with the handling of lists, so yes, might be good.